services/efs-guardian/src/simcore_service_efs_guardian/services/notifications/_notifier.py

```diff
@@ -0,0 +1,59 @@
+import contextlib
+from typing import Final
+
+import socketio  # type: ignore[import-untyped]
+from fastapi import FastAPI
+from fastapi.encoders import jsonable_encoder
+from models_library.api_schemas_webserver.socketio import SocketIORoomStr
+from models_library.projects_nodes_io import NodeID
+from models_library.users import UserID
+from pydantic import BaseModel, ByteSize, Field
+from servicelib.fastapi.app_state import SingletonInAppStateMixin
+
+SOCKET_IO_EFS_DISK_USAGE_EVENT: Final[str] = "efsNodeDiskUsage"
+
+
+class EfsNodeDiskUsage(BaseModel):
```

Comment:
As discussed, let's try to create a common models module for socketio interface. I suggests something like `models_library.socketio_schemas_webserver` and move there all the models used in the socketio API, including those already in `services/web/server/src/simcore_service_webserver/socketio/models.py`

Then in the future we can incorporate as well asyncapi specs https://www.asyncapi.com/en